js_modules/dagster-ui/packages/ui-core/src/ui/Filters/useStaticSetFilter.tsx

```diff
@@ -43,6 +44,7 @@ export function useStaticSetFilter<TValue>({
   menuWidth,
   allowMultipleSelections = true,
   matchType = 'any-of',
+  closeOnSelect = false,
```

Comment:
@salazarm I couldn't tell if there was already an idiomatic way to do this. Happy to change to that if there is one, lmk if I missed it.

Reply:
This looks good to me!

Reply:
Although, if we switch to checkboxes and allow multiple we likely won't need this? Maybe that should be the default behavior 🤔